tests/logger_test.go

```diff
@@ -0,0 +1,40 @@
+package tests
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/techrail/bark/db"
+	"github.com/techrail/bark/logger"
+)
+
+func TestGetLogger(t *testing.T) {
+	serviceName := "TestService"
+	bark := logger.GetLogger(serviceName)
+
+	weirdJsonData, _ := json.Marshal(map[string]interface{}{
+		"weird_key_1": "destroy the world",
+		"weird_key_2": "or may wait until tomorrow",
+	},
+	)
+
+	bark(0, "901101", "This logger rocks!", weirdJsonData)
+
+	database, err := db.ConnectToDatabase()
+
+	if err != nil {
+		t.Log("Failed to connect to database for verificaiton")
+		t.Fail()
+	}
+
+	logs, err := database.FetchLimitedLogs(100)
+
+	for i := 0; i < 100; i++ {
```

Comment:
This `100` is why we usually have a "test setup" and "test teardown". I will allow this for now. But we need to do the test buildup and teardown in some time.